firefox-ios/Client/AccessoryViewProvider.swift

```diff
@@ -53,10 +55,15 @@ class AccessoryViewProvider: UIView, Themeable {
     }()
 
     private lazy var doneButton: UIBarButtonItem = {
-        .init(title: .CreditCard.Settings.Done,
-              style: .done,
-              target: self,
-              action: #selector(tappedDoneButton))
+        let button = UIButton(type: .system)
+        button.setTitle(.CreditCard.Settings.Done, for: .normal)
```

Comment:
not necessarily related to credit card anymore, do we want to update to make it more generic?

Reply:
We could change the name of variable but we usually tie them to the string key, in this case "CreditCards.Settings.Done.v114". My only concern would be if the strings were already exported and we get translation for it, it would mean to start over with new name new key.